CalibCalorimetry/HcalAlgos/src/HcalDbHardcode.cc

```diff
@@ -784,11 +785,24 @@ void HcalDbHardcode::makeHardcodeFrontEndMap(HcalFrontEndMap& emap, const std::v
 }
 
 HcalSiPMParameter HcalDbHardcode::makeHardcodeSiPMParameter (HcalGenericDetId fId) {
+  if (fId.isHcalDetId()) {
+    if (fId.subdetId() == HcalBarrel) {
+      return HcalSiPMParameter(fId.rawId(), HcalHBHamamatsu1, 57.5, 0.055, 0, 0);
+    } else if (fId.subdetId() == HcalEndcap) {
+      return HcalSiPMParameter(fId.rawId(), HcalHEHamamatsu1, 57.5, 0.055, 0, 0);
+    } else if (fId.subdetId() == HcalOuter) {
+      return HcalSiPMParameter(fId.rawId(), HcalHOHamamatsu, 4.0, 0.055, 0, 0);
+    }
+  } 
   return HcalSiPMParameter(fId.rawId(), 0, 0, 0, 0, 0);
 }
 
 void HcalDbHardcode::makeHardcodeSiPMCharacteristics (HcalSiPMCharacteristics& sipm) {
-  sipm.loadObject(0,0,0,0,0,0,0,0);
+  sipm.loadObject(HcalHOZecotek,36000,1,0,0,0.32,0,0);
```

Comment:
@bsunanda - please add a few lines of documentation on what these numbers are/where they come from. Thanks!